getting-started/vs-integration/new-project-wizard.md

```diff
@@ -83,6 +82,18 @@ The Create New Project wizard provides the following project templates:
 * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.
 * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
+### Step 3: Configure Additional Project Settings
+
+This screen provides option to configure:
+
+   * Whether to import the required Telerik UI stylesheet and JS Interop file by suing the [static assets](/blazor-ui/getting-started/what-you-need#using-static-assets) or the [CDN method](/blazor-ui/getting-started/what-you-need#using-cdn).
+   * Whether to use [localization]({%slug globalization-localization%}) in the project or not.
+   * Whether to use [Font Icons]({%slug common-features-icons%})
```

Comment:
```suggestion
   * Whether to use [Font Icons]({%slug common-features-icons%}).
```

Reply:
I think I already updated that but I'll check.